src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+const showSuccessMessage = (message) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = message;
+  body.appendChild(div);
+};
+
+const showErrorMessage = (error) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message', 'error-message');
+  div.textContent = error.message;
+  body.appendChild(div);
+};
+
+promise1.then(showSuccessMessage).catch(showErrorMessage);
+promise2.then(showSuccessMessage).catch(showErrorMessage);
```

Comment:
Since promise2 will always reject after 3 seconds, consider whether this behavior is intended. If not, you might want to adjust the logic to handle different scenarios or conditions for rejection.